google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiSelectorPanel.java

```diff
@@ -224,6 +249,27 @@ private void updateManagementUI() {
     detailsPanel.setManagementUIEnabled(addLibrary && projectSelector.getSelectedProject() != null);
   }
 
+  /**
+   * Populates the BOM {@link JComboBox} with the fetched versions. If there are no versions
+   * returned, then the BOM UX is hidden.
+   */
+  // TODO (eshaul): make async with loader icons
+  private void populateBomVersions() {
+    List<Version> bomVersions =
+        CloudApiMavenService.getInstance().getBomVersions(NUM_BOM_VERSIONS_TO_SHOW);
+
+    if (bomVersions.isEmpty()) {
+      hideBomUI();
```

Comment:
So per your description, this will hide BOM versions and UI controls, and instead show individual versions and show them in library details? In this case, do we also plan to add individual versions to POM file until BOM is available, correct?

Reply:
yes. an empty return from the BOM service call will do the following:

1) hide the BOM label and dropdown (done in this PR)
2) in the client lib selector UI, show the versions from the metadata json instead of from the BOM (effectively already in place because fetching lib versions based on the BOM is not yet implemented)
3) add the dependencies to the pom.xml as if there is no BOM support at all (like we already do)